src/components/FloatingDropdown/FloatingDropdown.st.css

```diff
@@ -30,6 +30,10 @@
     -st-extends: FloatingDropdownBase;
 }
 
+.root .dropdown:content-visible::popoverContent {
+    background-color: "color(color-1)";
+}
+
```

Comment:
@kranux 
Is it working now?

Reply:
Still not working as expected. 

I think, its because I'm trying to reuse `.dropdown` imported from local `<Dropdown/>` component here 

https://github.com/wix/wix-ui-tpa/blob/22154ec07a9e9644c22314b32302c22fb5df24e7/src/components/FloatingDropdown/FloatingDropdown.st.css#L18